components/datadog/agent/ecsFargate.go

```diff
@@ -9,15 +9,11 @@ import (
 	"github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
 )
 
-func ECSFargateLinuxContainerDefinition(e config.Env, image string, apiKeySSMParamName pulumi.StringInput, fakeintake *fakeintake.Fakeintake, logConfig ecs.TaskDefinitionLogConfigurationPtrInput) *ecs.TaskDefinitionContainerDefinitionArgs {
-	if image == "" {
-		image = dockerAgentFullImagePath(e, "public.ecr.aws/datadog/agent", "latest", false)
-	}
-
+func ECSFargateLinuxContainerDefinition(e config.Env, apiKeySSMParamName pulumi.StringInput, fakeintake *fakeintake.Fakeintake, useLatestStableAgent bool, logConfig ecs.TaskDefinitionLogConfigurationPtrInput) *ecs.TaskDefinitionContainerDefinitionArgs {
```

Comment:
I think we can keep the `image` args to have something more flexible. And instead if we want to minimize changes we can change the next lines:
from
```
	if image == "" {
		image = dockerAgentFullImagePath(e, "public.ecr.aws/datadog/agent", "latest", false)
	}
```
to
```
	if image == "" {
		image = dockerAgentFullImagePath(e, "public.ecr.aws/datadog/agent", "", false)
	}
```
That way we will use the image built in the pipeline by default. And when we want to use latest we simply pass the latest image